client/my-sites/github-deployments/components/deployment-style/workflow-validation-wizard.tsx

```diff
@@ -46,6 +55,8 @@ export const WorkflowValidationWizard = ( {
 			);
 		}
 
+		dispatch( recordTracksEvent( 'calypso_hosting_github_workflow_valid' ) );
```

Comment:
We should wrap these calls on a `useEffect` call, otherwise these Tracks events will be dispatched on every render.